gemd/entity/attribute/base_attribute.py

```diff
@@ -49,7 +49,7 @@ def __init__(self,
         self._value = None
         self._template = None
         self._origin = None
-        self._file_links = None,
+        self._file_links = None
```

Comment:
Collateral discovery.  Just a cosmetic issue b/c the attribute is initialized 4 lines later.